GUIs/ScriptRunnerPyQt5_GUI/scripFunc/scripWINONLY.py

```diff
@@ -524,13 +549,195 @@ def calladdusrtosys():
+                    # print(a)
+                    for i in range(0, len(a2)):
+                        # print(a[i])
+                        x = a2[i]
+                        y = x[0]
+                        EXEC = sub.Popen(["powershell", "& {$env:UserName}"], stdout = sub.PIPE)
+                        stdout, _ = EXEC.communicate()
+                        output = stdout.decode("utf-8")
+                        output = output.split('\r\n')
+                        if output[0] == y:
+                            y = output[0] + '   (Current User)'
+                        # print(y)
+                        names.append(y)
+                    # print(names)
+                    return names
+
+                listoNames = findnames()
+
+                for i in range(0, len(listoNames)):
+                    QListWidgetItem(listoNames[i], self.listOFnames)
+
+                def removal():
+                    username = self.Username_Input.text()
+
+                    def completedPOP(username):
+                        COMPLETE = QMessageBox()
+                        COMPLETE.setIcon(QMessageBox.Question)
+                        COMPLETE.setWindowTitle('Hey! Listen!')
+                        COMPLETE.setText(
+                            'User ' + username + ' has been successfully removed from the system.')
+                        COMPLETE.setStandardButtons(QMessageBox.Close)
+                        COMPLETE.exec_()
+                        self.close()
+
+                    if len(username) == 0 or username not in listoNames:
+                        if len(username) == 0:
+                            print('no username entered')
+                            ERROR_NO_USER = QMessageBox()
+                            ERROR_NO_USER.setIcon(QMessageBox.Warning)
+                            ERROR_NO_USER.setWindowTitle('Hey! Listen!')
+                            ERROR_NO_USER.setText(
+                                'No username was entered. No user was removed.')
+                            ERROR_NO_USER.setStandardButtons(QMessageBox.Close)
+                            ERROR_NO_USER.exec_()
+                        else:
+                            print('User not found')
+                            ERROR_NO_USER_FOUND = QMessageBox()
+                            ERROR_NO_USER_FOUND.setIcon(QMessageBox.Warning)
+                            ERROR_NO_USER_FOUND.setWindowTitle('Hey! Listen!')
+                            ERROR_NO_USER_FOUND.setText(
+                                'User was not found on system. No user was removed.')
+                            ERROR_NO_USER_FOUND.setStandardButtons(QMessageBox.Close)
+                            ERROR_NO_USER_FOUND.exec_()
+                    else:
+                        print('User was removed............ This is a test')
+                        # Executes command.
+                        sub.Popen(["powershell", "& {net user " + username + " /DELETE}"])
```

Comment:
Codacy found an issue: [subprocess call - check for execution of untrusted input.](https://app.codacy.com/manual/vipersniper0501/CP_Scripts2/pullRequest?prid=5927260)